includes/admin/class-sensei-onboarding.php

```diff
@@ -46,6 +47,8 @@ public function __construct() {
 		if ( is_admin() ) {
 
 			add_action( 'admin_menu', [ $this, 'admin_menu' ], 20 );
+			add_action( 'admin_notices', [ $this, 'onboarding_wizard_notice' ] );
+			add_action( 'admin_init', array( $this, 'skip_setup_wizard' ) );
```

Comment:
We seem to be calling it both the "onboarding wizard" and the "setup wizard". Is this for legacy reasons? If not, could we make terminology more consistent? Probably "onboarding wizard" is what we want to call it, right?

Reply:
I think this is my bad. I was calling it the onboarding wizard but then I switched to setup wizard. Because we are using "setup wizard" in the UI, it's probably better to use that.

Reply:
Renamed for this PR here: https://github.com/Automattic/sensei/pull/3096/commits/4782bd94094f308d1cdddbb1351de4bd72e82ccc